lib/livebook/evaluator/default_formatter.ex

```diff
@@ -80,4 +80,15 @@ defmodule Livebook.Evaluator.DefaultFormatter do
       reset: :reset
     ]
   end
+
+  defp error_type(error) do
+    cond do
+      mix_install_vm_error?(error) -> :mix_install_vm_error
+      true -> :other
+    end
+  end
+
+  defp mix_install_vm_error?(exception) do
+    Exception.message(exception) =~ "Mix.install/2 can only be called with the same dependencies"
```

Comment:
Should we also check the exception struct is Mix.Error? 🤔

Reply:
Sounds good!